pkg/asset/manifests/gcp/cloudproviderconfig.go

```diff
@@ -51,7 +56,7 @@ func CloudProviderConfig(infraID, projectID, subnet string) (string, error) {
 	return buf.String(), nil
 }
 
-var configTmpl = `[global]
+var configTmpl = fmt.Sprintf(`[global]
```

Comment:
why wrap this with `fmt.Sprintf`? It seems like it is not needed.

Reply:
Oh that is a mistake that snuck back in